contracts/os/andromeda-kernel/src/execute.rs

```diff
@@ -351,13 +353,11 @@ fn handle_ibc_transfer_funds_reply(
             .collect::<Vec<String>>()
             .join("/");
 
-        adjusted_funds = Coin::new(
-            ics20_packet_info.funds.amount.u128(),
-            format!(
-                "ibc/{}/{}",
-                adjusted_path, _counterparty_denom_info.base_denom
-            ),
-        );
+        let denom_path = format!("{}/{}", adjusted_path, &ics20_packet_info.funds.denom);
```

Comment:
This entire codeblock should be unnecessary now

Reply:
Without the block we get this denom (which causes the integration test to fail): ibc/6A3FD85A1F1BFFDCE0DAA7E50E44B489FFCCB5E539306C207F532863BAA57046

With it we get:
ibc/1b2f8f2baf5b42370343270756f8180dd56acc9aa1699406df7821ae75608c99